assets/component-slider.css

```diff
@@ -61,13 +71,20 @@ slider-component {
     scroll-padding-left: 1.5rem;
     -webkit-overflow-scrolling: touch;
     margin-bottom: 1rem;
-    padding-top: max(0.5rem, calc(var(--shadow-vertical-offset) * -1 + var(--shadow-blur-radius)));
-    padding-bottom: max(0.5rem, calc(var(--shadow-vertical-offset) + var(--shadow-blur-radius)));
   }
 
   .slider.slider--tablet .slider__slide {
     margin-bottom: 0;
-    padding-bottom: 0;
+    padding-top: max(var(--focus-outline-padding), var(--shadow-padding-top));
+    padding-bottom: max(var(--focus-outline-padding), var(--shadow-padding-bottom));
+  }
+
+  .slider.slider--tablet.contains-card--standard .slider__slide:not(.collection-list__item--no-media) {
```

Comment:
Just a very general thought. I wonder if we should look into, at some point, at a refactor for class specificity so that we don't rely on `.class.class.class` as much but instead `:where(.class.class.class) .slider__slide:not...`
🤔 
`:where()` has a specificity of zero at all time. So basically it can help a bit with specificity of styling in general. Maybe Ill explore in some free time if it would be helpful/has a positive impact.